README.md

```diff
@@ -46,3 +46,10 @@ Thank you to the following contributors:
 - Yuvi Panda
 
 Future contributors are encouraged to add themselves to this README file too.
+
+## Licensing
+
+This repository is dual licensed under the Apache2 (to match upstream kubernetes
```

Comment:
to match *the* kubernetes charts repository.

to match *the* rest of Project Jupyter repositories